7_troubleshooting/01_common_issues.txt

```diff
@@ -109,7 +109,7 @@ database, it may take several minutes to a couple of hours
 
 ----
 # First, stop the Rudder server
-touch /opt/rudder/etc/disable-agent
+rudder agent disable
 /etc/init.d/rudder-jetty stop
```

Comment:
Since this is for 3.1, I think we should update this to be "service rudder-jetty stop" to be compatible with alternative init systems.